app/src/main/java/com/duckduckgo/app/browser/PulseAnimation.kt

```diff
@@ -30,6 +31,7 @@ import androidx.lifecycle.DefaultLifecycleObserver
 import androidx.lifecycle.LifecycleOwner
 import com.duckduckgo.mobile.android.ui.view.setAllParentsClip
 
+@SuppressLint("NoLifecycleObserver") // we don't use DI here
```

Comment:
The comment is kinda wrong, the justification is not that `we don't use DI`. Is that we're not observing the app lifecycle.
Same for other comments

Reply:
true. I was thinking about that when writing my previous comment 👍